src/common/fam_config_info.cpp

```diff
@@ -56,6 +56,18 @@ std::string yaml_config_info::get_key_value(std::string key) {
     return value;
 }
 
+std::string yaml_config_info::get_map_value(std::string memservers, uint64_t memserver_id, std::string attr) {
```

Comment:
In this function can you give variable like - name, id and attr instead of specific names ?